### PR TITLE
Channels API

### PR DESCRIPTION
**Pull request by shivamag00**
Review Requested: @jrobbins 

In this PR,

- added API for channels.

`schedule.html` requires feattures list and channels with release schedule. This PR creates an API for channels. In a future PR, we will try to use this API for fetching the channels.

### CODE REVIEW

File: api/channels_api_test.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import mock
+import unittest
+
+from api import channels_api
+
+
+class ChannelsAPITest(unittest.TestCase):
+
+  @mock.patch('api.channels_api.fetch_chrome_release_info')
+  @mock.patch('api.channels_api.fetchchannels.get_omaha_data')
```
Comment (shivamag00) [inline api/channels_api_test.py:28]: I have patched objects in the namespace where they are used.

**Review by shivamag00 — COMMENTED** [review]
@jrobbins, I have never used `@mock.patch()` before. Please let me know if I have used it incorrectly.

File: api/channels_api_test.py (hunk 1 above)
Comment (jrobbins) [reply api/channels_api_test.py:28]: Hmm, I would expect it to be
@mock.patch('internals.fetchchannels.get_omaha_data')
If you see tests passing, then I suppose it works either way, but I think specifying the location where a function is defined is more conventional.

**Review by jrobbins — CHANGES_REQUESTED** [review]

File: api/channels_api_test.py (hunk 1 above)
Comment (shivamag00) [reply api/channels_api_test.py]: Yes the test was a success. Anyway, I have changed it to what you suggested thanks.

**Review by jrobbins — APPROVED** [review]

**Pull request merged**